components/Layout/Footer.tsx

```diff
@@ -0,0 +1,93 @@
+import Link from 'next/link';
+import { hoverClass } from 'styles/globals';
+
+export const Footer = () => (
+  <footer className="footer bg-black">
+    <div className="grid grid-cols-3 grid-rows-3 max-sm:grid-cols-1 gap-x-8 text-white p-10 max-w-screen-2xl xl:w-[1400px] mx-auto">
+      <div className="md:row-start-2">
+        <span className="text-5xl">Valory</span>
+      </div>
+
+      <div className="md:col-start-2 flex lg:flex-row flex-col gap-x-8">
+        <Link className={hoverClass} href="#">
+          Mission Statement
+        </Link>
+        <Link className={hoverClass} href="#">
+          Get involved
+        </Link>
+
+        <details className="dropdown">
```

Comment:
can you please check how to support the click outside logic? here's an example I think https://daisyui.com/components/dropdown/#dropdown-menu